libs/openchallenges/home/src/lib/challenge-search/challenge-search.component.ts

```diff
@@ -81,6 +84,12 @@ export class ChallengeSearchComponent implements OnInit {
         pageSize: 1,
       })
       .pipe(map((page) => page.totalElements));
+    this.platformCount$ = this.challengePlatformService
+      .listChallengePlatforms({
```

Comment:
Is there a way we can filter out platforms with a challenge count of 0?  We currently don't list CASP, OpenML, or. PapersWithCode on the [homepage](https://www.figma.com/file/kjlOBxTQpJY79hUju4LVUT/OC-Platforms?type=design&node-id=0-1&mode=design&t=OnoN0k5KS2RqICXu-0).

Reply:
OR - if this makes more sense - I can add a flag to the platforms page on whether we display the platform on the app or not.  This would be similar to what we did for organizations.

Reply:
@tschaffter @rrchai @gaiaandreoletti

Reply:
I'm in favor of the second option, i.e. export to CSV only the platforms that should be exported. We could add `visibility` property in the future so that all the data are stored in the DB but that would require work to be planned to update the backend.

Reply:
> I'm in favor of the second option,

Okay, same.  This change will be done with the db update tomorrow.